src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    static ColorSupplier colorSupplier = new ColorSupplier();
+    public static Ball getRandomBall(){
```

Comment:
The method `getRandomBall` should not be static. Let's create an instance of the `Lottery` class in the `Application` class and use this instance to call `getRandomBall`.